rest/src/main/kotlin/builder/channel/thread/ThreadModifyBuilder.kt

```diff
@@ -31,8 +31,8 @@ public class ThreadModifyBuilder : AuditRequestBuilder<ChannelModifyPatchRequest
     private var _invitable: OptionalBoolean = OptionalBoolean.Missing
     public var invitable: Boolean? by ::_invitable.delegate()
 
-    private var _flags: Optional<ChannelFlags> = Optional.Missing()
-    public var flags: ChannelFlags? by ::_flags.delegate()
+    private var _appliedTags: Optional<List<Snowflake>> = Optional.Missing()
+    public var appliedTags: List<Snowflake>? by ::_appliedTags.delegate()
```

Comment:
These should be `MutableList`s to make adding to them more simple for adding too.